### PR TITLE
Fix class prop test fixture

### PR DESCRIPTION
**Pull request by existentialism**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues             | n
| Patch: Bug Fix?          | n
| Major: Breaking Change?  | n
| Minor: New Feature?      |  n
| Tests Added/Pass?        | y/y
| Spec Compliancy?         | n
| License                  | MIT
| Doc PR                   | 
| Any Dependency Changes?  | 

Updating test fixtures from https://github.com/babel/babel/pull/6082 due to changes from https://github.com/babel/babel/pull/6076.

(As @hzoo predicted :D)

### CODE REVIEW

File: packages/babel-plugin-transform-class-properties/test/fixtures/loose/constructor-collision-ignores-types/expected.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError("Cannot call a class as a function"); } }
```
Comment (andy-ms) [inline packages/babel-plugin-transform-class-properties/test/fixtures/loose/constructor-collision-ignores-types/expected.js]: The original version of this test compiled to a class -- I'd like to keep that behavior as I'm not sure if the issue reproduces when compiling all the way down to a function.

Comment (existentialism) [inline packages/babel-plugin-transform-class-properties/test/fixtures/loose/constructor-collision-ignores-types/expected.js]: @andy-ms updated!

Comment (andy-ms) [inline packages/babel-plugin-transform-class-properties/test/fixtures/loose/constructor-collision-ignores-types/expected.js]: Thanks!

**Comment by Kovensky** [comment]
I should have looked in here before poking master.

I wonder why only one of the fixtures in master was affected, instead of both like in here.

**Comment by existentialism** [comment]
@Kovensky hah, I actually added another fixture for loose mode

**Comment by existentialism** [comment]
Half of this was landed in https://github.com/babel/babel/commit/cd66657f347ce132957a296c7dd7435b581c62c0, will land just the loose mode fixture

**Pull request merged**